tests/rbd/test_rbd_live_migrate_encryption.py

```diff
@@ -101,6 +102,42 @@
 12. Unmap the disk and cleanup the pools, namespace and image
 13. Repeat the test on LUKS2
 
+Test case covered:
```

Comment:
It's not good to add other test case also in the existing test, can we keep separate module for this test case?